common/src/main/resources/parserFieldValidValues.txt

```diff
@@ -1,14 +1,14 @@
 LEI|String with only letters and numbers
 ULI|String with only letters and numbers
 LAR Record Identifier|Integer: 2
-Application Date|Integer or NA
+Application Date|Valid Date in format yyyyMMdd or NA
 Loan Type|Integer: 1, 2, 3, 4
 Loan Purpose|Integer: 1, 2, 31, 32, 4, 5
 Preapproval|Integer: 1, 2
 Construction Method|Integer: 1, 2
 Occupancy Type|Integer: 1, 2, 3
 Action Taken|Integer: 1, 2, 3, 4, 5, 6, 7, 8
-Action Taken Date|Integer
+Action Taken Date|Valid Date in format YYYYMMDD
```

Comment:
@KennethGudel  are we sure these are the correct date formate regex i.e. yyyy vs YYYY

Reply:
This text is only used for the error message that is displayed to users. @chynnakeys suggested that using all caps would be less confusing to users.